src/Sylius/Behat/Page/Checkout/CheckoutThankYouPage.php

```diff
@@ -38,6 +38,16 @@ public function hasThankYouMessageFor($name)
     }
 
     /**
+     * @throws \InvalidArgumentException
+     */
+    public function wait()
```

Comment:
I think we should make the time an argument here, because I don't want to guess how long it will wait. ;)